mysql-test/t/buffered_error_log.test

```diff
@@ -0,0 +1,21 @@
+call mtr.add_suppression("Attempting backtrace. You can use the following information to");
```

Comment:
@dutow why did you suppress this? Does it mean there was assertion failure and core file generated?

Reply:
The test intentionally crashes the server.

Reply:
simple, kill_mysqld.inc is not enough?

Reply:
It's not, it doesn't suppress anything. You can see this suppression in many tests that use it.

Reply:
I mean, instead of send_kill_to_mysqld.inc, why not use kill_mysqld.inc which sends signal 9?

Reply:
ok, we can want to check if error log is saved to disk on core file generation/assert failures. Got it